_posts/2018-04-09-synching-sia.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Syncing Sia in 73 minutes
+layout: post
+date: "2018-04-09"
+description: "An experiment to see how fast I can perform the initial Sia blockchain sync"
+tags:
+- sia
+- blockchain
+- benchmark
+permalink: "/sia-sync-73-min/"
+---
+
+*This is a guest post by Sia community contributor [Thomas Bennett](https://github.com/tbenz9).*
+
+I can sync Sia from a block height of 0 to 149000 in 73 minutes.  Here’s how I do it.
```

Comment:
I feel like the intro is a bit abrupt. Can we ease the reader into it a bit more by explaining the context? We shouldn't assume they've read the previous blog post.

Why should the reader be interested and what's going to prevent them from immediately clicking the back button?

Suggest talking about:

* Motivations for why someone would want to do this
* Tell them you're going to teach them how to use this to speed up their own sync.

Reply:
Sure, I noticed it was a bit abrupt also, I hate when blogs spend too much time introducing the article, but I'll add a sentence or two.